content/applications/finance/accounting/taxes/retention.rst

```diff
@@ -2,68 +2,43 @@
 Withholding taxes
 =================
 
-A withholding tax, also called a retention tax, is a government
-requirement for the payer of a customer invoice to withhold or deduct
-tax from the payment, and pay that tax to the government. In most
-jurisdictions, withholding tax applies to employment income.
+A **withholding tax**, or retention tax, is a government requirement for the payer of a
+customer invoice to withhold or deduct tax from the payment and pay that tax to the government. In
+most jurisdictions, withholding tax applies to employment income.
 
-With normal taxes, the tax is added to the subtotal to give you the
-total to pay. As opposed to normal taxes, withholding taxes are deducted
-from the amount to pay, as the tax will be paid by the customer.
-
-As, an example, in Colombia you may have the following invoice:
-
-.. image:: retention/retention03.png
-   :align: center
-
-In this example, the **company** who sent the invoice owes $20 of taxes to
-the **government** and the **customer** owes $10 of taxes to the **government**.
+Usually, the tax is added to the subtotal to give the total amount to pay. In the case of
+withholding taxes, they are deducted from the amount to pay as the customer pays the tax.
 
 Configuration
 =============
 
-In Odoo, a withholding tax is defined by creating a negative tax. For a
-retention of 10%, you would configure the following tax (accessible
-through :menuselection:`Configuration --> Taxes`):
+In Odoo, a withholding tax is defined by creating a negative tax. To create a retention of 10%, go
```

Comment:
Is 10% a standard? If not, I would maybe generalize this sentence, and then add the 10% as an example, with the 2 screenshots. This would also avoid having two screenshots with a sentence sandwiched (and a little bit lost) in between, then another one sandwiched between the screenshot and the tip.  Let's discuss it :)